app/components/export-tool.js

```diff
@@ -82,44 +82,54 @@ export default Ember.Component.extend({
     _convertToJSON(dataArray) {
         return JSON.stringify(dataArray, null, 4);
     },
-    _convertToTSV(dataArray) {
+    _convertToCSV(dataArray) {
         // Flatten the dictionary keys for readable column headers
         let squashed = dataArray.map((item => squash(item)));
 
         var fields = Object.keys(squashed[0]);
-        var tsv = [fields.join('\t')];
+        var csv = [fields.join(',')];
```

Comment:
Objects are a hash, and records may not all have the same data. Make sure that the correct fields from each record line up from row to row. 

(don't just assume that the second field is always known)

Eg, how python does it:
https://docs.python.org/2/library/csv.html#csv.DictWriter

Getting the list of field names across all records will also be a challenge to address.